package.json

```diff
@@ -41,7 +41,7 @@
     "@types/google-protobuf": "^3.2.7",
     "@types/jquery": "^3.2.16",
     "@types/mocha": "^2.2.44",
-    "@types/node": "^8.0.53",
+    "@types/node": "^8.0.56",

```

Comment:
This update includes my pull request to update the constants in the type definition.